app/app.js

```diff
@@ -96,11 +105,25 @@ app.use(helmet.ieNoOpen())
 app.use(helmet.hidePoweredBy({setTo: 'Whisky Powered.'}))
 app.use(helmet.contentSecurityPolicy({
     directives: {
-        defaultSrc: ["'self'", "https://stackpath.bootstrapcdn.com"],  // default value for all directives that are absent
-        scriptSrc: ["'self'", "https://code.jquery.com/", "https://stackpath.bootstrapcdn.com", "https://cdnjs.cloudflare.com"],   // helps prevent XSS attacks
+        defaultSrc: [
+"'self'",
```

Comment:
The intendation is very messy here. it's hard to read it

Reply:
My bad, linter messed that up.